mkosi/config.py

```diff
@@ -61,10 +61,14 @@ class Verb(StrEnum):
     burn          = enum.auto()
 
     def supports_cmdline(self) -> bool:
-        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.ssh, Verb.journalctl, Verb.coredumpctl, Verb.burn)
+        return self in (
+            Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.ssh, Verb.journalctl, Verb.coredumpctl, Verb.burn
+        )
 
     def needs_build(self) -> bool:
-        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.serve, Verb.journalctl, Verb.coredumpctl, Verb.burn)
+        return self in (
+            Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.serve, Verb.journalctl, Verb.coredumpctl, Verb.burn
+        )
```

Comment:
With one-per-line this would be more readable…

Reply:
Playing around with `ruff format` I saw that even with a 119 column limit it would format this with one item per line, so I switched to that.